src/Common/Logging/SystemLogger.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+namespace OpenEMR\Common\Logging;
+
+use Monolog\Handler\ErrorLogHandler;
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
+use Waryway\PhpTraitsLibrary\Singleton;
+
+/**
+ * Class SystemLogger logs information out to the syslog and is a compatabile PSR3 logger.
+ * Other loggers can be added  here as needed.  We essentially decorate around the Monolog library
+ * but it allows us to remove Monolog if needed in the future, or add additional loggers as needed.
+ * @package OpenEMR\Common\Logging
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2020 Stephen Nielson <stephen@nielson.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+class SystemLogger implements LoggerInterface
+{
+    use Singleton;
+
+    /**
+     * @var LoggerInterface;
+     */
+    private $logger;
+
+    public function __construct()
+    {
+        /**
+         * We use mono
+         */
+        $this->logger = new Logger('OpenEMR');
+        $logLevel = Logger::DEBUG; // change this if you want to filter what logs you see.
```

Comment:
Is this set to what should be for production?

Reply:
I'll change it to warning.